src/vos/tests/evt_ctl.sh

```diff
@@ -159,9 +159,9 @@ echo "$cmd"
 $cmd -t
 
 if [[ -z ${CMOCKA_MESSAGE_OUTPUT+x} || -z ${CMOCKA_XML_FILE+x} ]]; then
```

Comment:
In my tests here, this isn't really valid syntax.  I'm not sure why you have a [ ] surrounding the two tests.  I think what you really want is:

```bash
if [ -z ${CMOCKA_MESSAGE_OUTPUT+x} ] || [ -z ${CMOCKA_XML_FILE+x} ]; then
```

Also, what does the `+x` inside the `${...}` variable expansions do?

Reply:
If CMOCKA_MESSAGE_OUTPUT or CMOCKA_XML_FILE is not set, the test will evaluate to x.

Reply:
That's not my experience:

```bash
$ unset foobar
$ echo ${foobar+x}

```

Also, if it did evaluate to `x` when would the `-z` test ever be true?

Reply:
-bash-4.2$ unset foo
-bash-4.2$ ./test.sh
not defined
-bash-4.2$ export foo="xml"
-bash-4.2$ ./test.sh
defined
-bash-4.2$ cat test.sh
if [ -z "${foo+x}" ]
then
    echo "not defined"
else
    echo "defined"
fi
-bash-4.2$

Reply:
Sure.  But what does the `+x` do to change the result of that test?  I.e. if you remove the `+x` from the `-z "${foo+x}"` test is the result any different?

Reply:
Yes. both seems to work same. Ok. Removing the x.